src/forms/numberInput.test.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import NumberInput, {DECREMENT_BTN_CLASS, INCREMENT_BTN_CLASS} from './NumberInput';
+
+describe('NumberInput', function() {
+	const onChange = jest.fn();
+	const LABEL_TEXT = 'Are you bringing any guests?',
+		VALUE = 2,
+		NAME_ATTR = 'amountCount',
+		ERROR_TEXT = 'Not so fast. You have an error.',
+		PLACEHOLDER = 'Guests',
+		MAX_ATTR = 10,
+		MIN_ATTR = 1,
+		STEP_ATTR = 1;
+
+	let numberInputComponent,
+		inputEl;
+
+	beforeEach(() => {
+		const formAttrs = {
+			id: NAME_ATTR,
+			min: MIN_ATTR,
+			max: MAX_ATTR,
+			error: ERROR_TEXT,
+			required: true,
+		};
+		numberInputComponent = TestUtils.renderIntoDocument(
+			<NumberInput
+				name={NAME_ATTR}
+				label={LABEL_TEXT}
+				value={VALUE}
+				{...formAttrs}
+			/>
+		);
+
+		inputEl = TestUtils.findRenderedDOMComponentWithTag(numberInputComponent, 'input');
+	});
+
+	afterEach(() => {
+		numberInputComponent = null;
+		inputEl = null;
+	});
+
+	it('exists', () => {
+		expect(() => TestUtils.findRenderedDOMComponentWithTag(numberInputComponent, 'input')).not.toThrow();
```

Comment:
Hmm we should be testing that the component instance executed correct here for an exists test, what you have is a good thing to test also but our exists test should really be an instance test :)

which would be 
`expect(() => TestUtils. findRenderedComponentWithType(numberInputComponent, NumberInput)).not.toThrow();`